cmd/cosign/cli/sign_blob.go

```diff
@@ -146,3 +145,16 @@ func SignBlobCmd(ctx context.Context, keyPath, kmsVal, payloadPath string, b64 b
 	}
 	return signature, nil
 }
+
+func sign(ctx context.Context, keyPath string, payload []byte, pf cosign.PassFunc) (signature []byte, publicKey *ecdsa.PublicKey, err error) {
+	k, err := loadKey(keyPath, pf)
+	if err != nil {
+		return
```

Comment:
I won't complain tooooo hard but I personally don't love the naked returns. @priyawadhwa likes them too so I'm outnumbered.

Reply:
clothed the returns

Reply:
honestly I don't love them either I think I did that back when I was returning like four things & thought it might be easier for someone new to understand